src/components/EncounterPage/EncounterPage.js

```diff
@@ -1,19 +1,39 @@
-import React from 'react'
+import React, { useState } from 'react'
 import EnemyEntry from './EnemyEntry'
 import PlayerEntry from './PlayerEntry'
 import PageButton from '../Buttons/PageButton'
-import styled from 'styled-components'
+import styled from 'styled-components/macro'
+import TurnButton from '../Buttons/TurnButton'
 
 export default function EncounterPage({ creatureEntries }) {
-  const creaturesByInitiative = creatureEntries
+  const [activeIndex, setActiveIndex] = useState(0)
+  const [turnNumber, setTurnNumber] = useState(1)
+  const [roundNumber, setRoundNumber] = useState(1)
+
+  const players = creatureEntries
+    .slice()
+    .filter((creatureEntry) => creatureEntry.playerName)
+  const enemies = creatureEntries
+    .slice()
+    .filter((creatureEntry) => creatureEntry.enemyName)
+  const playersToSort = players.map((player) => ({
+    ...player,
+    initiative: player.playerInitiative,
+  }))
+  const creaturesToSort = [...playersToSort, ...enemies]
+  const creaturesByInitiative = creaturesToSort
     .slice()
-    .sort((entry1, entry2) => entry1.initiative < entry2.initiative)
+    .sort(
+      (entry1, entry2) => Number(entry1.initiative) < Number(entry2.initiative)
```

Comment:
You are sorting with "less than". This returns a boolean. The sort function should return a number, see MDN: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
Better subtract one intitative value from the other and return the result.

Reply:
Makes sense. Thanks for the suggestion, I updated the function :)